include/nbl/asset/material_compiler/CMaterialCompilerGLSLBackendCommon.h

```diff
@@ -26,12 +26,14 @@ struct nbl_glsl_MC_emitter_t
 {
 	uvec2 emissive;
 	uvec2 emissionProfile;
-	float normalizeEnergy;
-	float orientation[6]; // not vector because of alignment, [0:3] left [3:6] up
+	float orientation[6]; // LSB of orientation[0] is 1 if right handed; [0:3] y axis [3:6] z axis
```

Comment:
so [0,3] is y-axis, why did you call it "left" before?

Reply:
it was x axis and y axis before but I changed it cause you asked one of them to be z.

Reply:
ok. I remember, cause isotropic gives no fs about X and Y, only Z